### DISCUSSION TITLE
Chaos Proxy Server: Delay set to 0 results in 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by anderswassen** [EyevinnOSC/community#35] [Q&A] [answered]
Hi!

I am using the Chaos Stream Proxy and everything works as a charm except one corner case... I am setting up a script to generate different "chaos profiles" and realized that this request results in a 550 Internal Server Error ({"reason":"Unexpected end of JSON input"}):
https://awatenant-chaosproxyawa.eyevinn-chaos-stream-proxy.auto.prod.osaas.io/api/v2/manifests/hls/proxy-master.m3u8?url=https://maitv-vod.lab.eyevinn.technology/VINN.mp4/master.m3u8&delay=[{i:*,ms:0}]

Ie, to set the delay to 0 ms makes the Chaos Proxy to fail. I understand that this particular corner case is somewhat useless but I would like to compare the delay I get using the Chaos Proxy vs not using the Chaos Proxy. I also would like to script the generation of "chaos" where I can choose a delay between 0-X (not being forced to have at least 1 ms i delay).

Keep up the good work!

**Comment by birme** [comment, marked as answer]
Hi,

Sound like a really interesting project you have got going!

While this question more belongs to the maintainer of the Chaos Stream Proxy project I took the liberty and posted this request in the Chaos Stream Proxy project with a reference to this thread.

Issue: https://github.com/Eyevinn/chaos-stream-proxy/issues/70
